templates/dashboard-tests/overlay/tmp/run.sh

```diff
@@ -166,6 +166,9 @@ if [ ${CORRAL_rancher_type} = "existing" ]; then
       -e TEST_SKIP=setup \
       -e AWS_ACCESS_KEY_ID=${CORRAL_aws_access_key} \
       -e AWS_SECRET_ACCESS_KEY=${CORRAL_aws_secret_key} \
+      -e AZURE_CLIENT_ID=${CORRAL_azure_client_id} \
+      -e AZURE_CLIENT_SECRET=${CORRAL_azure_client_secret} \
+      -e AZURE_AKS_SUBSCRIPTION_ID=${CORRAL_azure_subscription_id} \
```

Comment:
Are these inputs already declared somewhere, in the template manifest or the package itself? or they don't even need that?